dll/win32/shell32/CDefView.cpp

```diff
@@ -1582,6 +1582,18 @@ HRESULT CDefView::InvokeContextMenuCommand(CComPtr<IContextMenu>& pCM, LPCSTR lp
         cmi.ptInvoke = *pt;
     }
 
+    WCHAR szDirW[MAX_PATH];
+    CHAR szDirA[MAX_PATH];
+    szDirW[0] = 0;
```

Comment:
```suggestion
    WCHAR szDirW[MAX_PATH] = L"";
    CHAR szDirA[MAX_PATH];
```

Reply:
`szDirW[0] = 0;` is a single instruction, `[MAX_PATH] = L"";` is a function call to `memset`!

Reply:
Agree to @whindsaks.